zerver/lib/email_mirror.py

```diff
@@ -105,7 +106,7 @@ def create_missed_message_address(user_profile, message):
     if settings.EMAIL_GATEWAY_PATTERN == '':
         logging.warning("EMAIL_GATEWAY_PATTERN is an empty string, using "
                         "NOREPLY_EMAIL_ADDRESS in the 'from' field.")
-        return settings.NOREPLY_EMAIL_ADDRESS
+        return "Zulip <%s>" % (FromAddress.NOREPLY,)
```

Comment:
I think this should just be `FromAddress.NOREPLY`? I think it just ends up here around line 331 in notifications.py (though you should verify):
```
        'reply_to_email': address,
```
It would be nice to do something more clever here (e.g. put the name of the person you're replying to, or "Zulip", or something, but that would have to happen in `do_send_missedmessage_events_reply_in_zulip` rather than here.